web/massastation/src/pages/Store/StationSection/StationSection.tsx

```diff
@@ -39,6 +39,17 @@ function StationSection({
     }
   }, [error, navigate]);
 
+  // Sorting the plugins array alphabetically by plugin name
+  const sortedPlugins = plugins?.sort((a, b) => {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
+  });
```

Comment:
What we are sorting here!?
Object, array of strings or numbers!? 🤔 

Isn't that enought?
```ts
my-object.sort((a, b) => a.age - b.age);
```

Reply:
an array of ojbects

Reply:
> What we are sorting here!? Object, array of strings or numbers!? thinking
> 
> Isn't that enought?
> 
> ```ts
> my-object.sort((a, b) => a.age - b.age);
> ```

I get this error with this suggestion, I do not think it is worth converting type to achieve this

The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.ts(2362)

Reply:
what if you parse int ?

Reply:
we found  a more efficient way :)